profile_report.py

```diff
@@ -0,0 +1,31 @@
+from profiler import data
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("benchmark", type=str)
+parser.add_argument("path", type=str)
+
+args = parser.parse_args()
+
+# Processes the set of previously written logs
+
+threshold = 1.10  # Percent difference
+
+db = data.FileBasedProfileDB(args.path)
+actual_max_ts = 0
+dt = db.find(f"{args.benchmark}")
+c = 0
+
+if len(dt) >= 2:
+    exec_time = [0.0, 1.0]
+    last_two = dt[-2:]
+    exec_time[0] = last_two[0].user_time_sec
+    exec_time[1] = last_two[1].user_time_sec
+
+    if threshold * float(exec_time[1]) < float(exec_time[0]) or float(exec_time[1]) > threshold * float(exec_time[0]):
+        raise SystemExit(f"Potential performance degradation detected {exec_time[0]} vs {exec_time[1]}")
```

Comment:
When a performance problem is detected, is there more information that could be output which is useful to the reader? Maybe just print out both of the `ProfileData` objects, since that's the probably the very next thing the reader would want to see.

Reply:
Great suggestion